Beta_Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer_description.md

```diff
@@ -1,4 +1,4 @@
- Before you can use AWS Access Analyzer, you need to perform several configuration steps in your AWS environment.
+Before you can use AWS Access Analyzer, you need to perform several configuration steps in your AWS environment.
```

Comment:
what was incorrect here? Just the extra space?

Reply:
Correct. Because of the way the end YAML is generated (https://github.com/demisto/content/blob/master/package_creator.py#L154-L173), the content is inserted as-is to the multiline string. This results in the following content:
```yaml
detaileddescription: |
   Before you can use AWS Access Analyzer, you need to perform several configuration steps in your AWS environment.
  
  ### Prerequisites
  - Attach an instance profile with the required permissions to the Demisto server or engine that is running on your AWS environment.
  - Instance Profile requires minimum permission: sts:AssumeRole.
  (...)
```

This is an invalid YAML because of invalid indentation in the block text (it assumes 3 spaces because of the first line, but then finds a 2-space line -- see https://yaml-multiline.info/). Multiple YAML parsers complain about it.

An alternative is to include a space indicator in the generated YAML:
```yaml
detaileddescription: |2
   Before you can use AWS Access Analyzer, you need to perform several configuration steps in your AWS environment.
  
  ### Prerequisites
  - Attach an instance profile with the required permissions to the Demisto server or engine that is running on your AWS environment.
  - Instance Profile requires minimum permission: sts:AssumeRole.
  (...)
```